public/js/searchList.js

```diff
@@ -0,0 +1,79 @@
+let searchCardsTemplate;
+
+/*
+ * When the page loads, fetch initial organization data and render it
+ * in cards on the list.
+ */
+window.onload = function() {
+  fetchOrganizations("all").then(organizations => {
+    createOrganizationCards(organizations);
+    createMarkers(organizations);
```

Comment:
You could call `createMarkers` in `maps.js`. It seems searchList logic is separate from the logic of the map. Calling `createMarkers` in `maps.js` would ensure that all logic in `searchList` deals with the searchList and doesn't touch the map.

Reply:
How would you recommend going about that? `createMarkers` uses the organization data, so it needs to wait for the organization data from the fetch before creating markers.

Reply:
That makes sense. Yeah that's the tricky part. My suggestions are not best practice. One way could be to make the organizations a global variable so that maps.js and searchList.js can access it.Or do another fetch in maps.js but thats just repeating and it doesn't doesn't make much sense. 

Maybe @brucezheng and @ccyang314 can give there input on this comment. The structure may be find for now.

Reply:
You could achieve this using some kind of event triggering, but I don't think it's necessary for now.